docs/usage/customization-guide.md

```diff
@@ -601,8 +601,9 @@ The following features are available for matching:
 |                  |              | **`status`** | string | Status of the driver, possible values are 'active' and 'passive'
 |                  |              | **`turbo`**  | bool   | 'true' if turbo frequencies are enabled, otherwise 'false'
 |                  |              | **`scaling`** | string | Active scaling_governor, possible values are 'powersave' or 'performance'.
-| **`cpu.rdt`**    | flag         |          |            | Intel RDT capabilities supported by the system
+| **`cpu.rdt`**    | attribute    |          |            | Intel RDT capabilities supported by the system
 |                  |              | **`<rdt-flag>`** |    | RDT capability is supported, see [RDT flags](#intel-rdt-flags) for details
+|                  |              | **`RDTL3CA_NUM_CLOSID`** | int  | The number or available CLOSid(Class of service ID) for Intel L3 Cache Allocation Technology
```

Comment:
nit: add space, I'd suggest to write CLOSID in all caps

```suggestion
|                  |              | **`RDTL3CA_NUM_CLOSID`** | int  | The number or available CLOSIDs (Class of service ID) for Intel L3 Cache Allocation Technology
```

Reply:
I have based this acronym from this document https://docs.kernel.org/x86/resctrl.html, they use `CLOSid` everywhere. But I have no strong opinion about that.

Reply:
changed.

Reply:
>  they use CLOSid everywhere. But I have no strong opinion about that.

OK, not a big deal. Took a look and they actually use CLOSID, CLOSid and closid at least so 🤷